transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/TransformTestWizardPage.java

```diff
@@ -229,7 +229,7 @@ public void widgetSelected(final SelectionEvent event) {
         Dependency dep = new Dependency();
         dep.setGroupId("org.apache.camel"); //$NON-NLS-1$
         dep.setArtifactId("camel-test-blueprint"); //$NON-NLS-1$
-        dep.setVersion(Util.getCamelVersion(project));
+        dep.setVersion(Util.camelVersion(project));
```

Comment:
why do u remove the "get" in getter method names? imho using "get" as prefix is the standard way.